cmd/ci-operator/main.go

```diff
@@ -1316,6 +1321,30 @@ func (o *options) initializeNamespace() error {
 	return nil
 }
 
+func generateAuthorAccessRoleBinding(namespace string, authors []string, mappingLoadFunc func() map[string]string) *rbacapi.RoleBinding {
```

Comment:
Do we get anything out of passing in the `mappingLoadFunc` vs just passing in the mappings?

Reply:
Updated.